roles/deprecated_api/README.md

```diff
@@ -25,26 +18,33 @@ The role requires the following:
 - Ensure the availability of the K8s [APIRequestCount](https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/bindata/assets/kube-apiserver/apiserver.openshift.io_apirequestcount.yaml) object. For OpenShift, APIRequestCount is included in the standard distribution from OCP 4.9 onwards. For other distributions, deploy the required CRD and CR before using the role.
 - Python version 3.6 or higher is needed to parse the APIs.
 
-
 # Parameters
 
 | Parameter               | Choices/Defaults      | Comments                                             |
 |-------------------------|-----------------------|------------------------------------------------------|
 | `kubeconfig`            | `~/.kube/config.json` | Path to an existing Kubernetes config file to be used by k8s_info dependency. If not provided, the OpenShift client will attempt to load the default configuration file from `~/.kube/config.json`. Can also be specified via `K8S_AUTH_KUBECONFIG` environment variable.                                                                                                |
-| `deprecated_namespaces` | `undefined`           | List of namespaces the role will check for deprecated APIs. Typically, it should include the namespaces where you deploy the workload. If left undefined, the role will check all namespaces, excluding those starting with the `openshift` and `kube-` prefixes.                                                |
+| `deprecated_namespaces` | `undefined`           | Optional. List of namespaces the role will check for deprecated APIs. Typically, it should include the namespaces where you deploy the workload. If left undefined, the role will check all namespaces, excluding those starting with the `openshift` and `kube-` prefixes.                                                |
 | `ocp_version`           | `"4.9"/"4.10"/etc`    | OpenShift version currently running on your cluster. |
-| `job_logs`              | `undefined`           | Directory to store the output JUnit file containing the 'Workload Compatibility with OCP Versions' testsuite.                                                              |
+| `deprecated_api_logs`   | `{{ playbook_dir }}`  | Optional. Directory to store the output JUnit file containing the 'Workload Compatibility with OCP Versions' testsuite.                                                              |
```

Comment:
In fact, the parameter is `deprecated_api_logs.path`

Reply:
Yes, I provide an example in `readme/# examples` so I believe that's ok